### DISCUSSION TITLE
Please return with the colored badge HTTP terms

### DISCUSSION THREAD

**Discussion by ChrisCoy** [Kong/insomnia#6373] [Ideas] [answered]
The old style is way better i think, it was so much easier to see the type of the route just by the badge color. And to those with bad vision is hard to read the tiny letters, please, return to the old style. Love you guys and love Insomnia 💜.

![image](https://github.com/Kong/insomnia/assets/62861341/36976695-e26f-4b81-9f3a-2f3d6df1b628)

**Comment by carmolim** [comment, marked as answer]
The same as https://github.com/Kong/insomnia/discussions/6371
